### PR TITLE
Model 0 Demography Implementation

### PR DESCRIPTION
**Pull request by hussain-jafari**
### Description
- *Category*: implementation
- *JIRA issue*: 
- *Research reference*: {IV Iron - Children under five}(https://vivarium-research.readthedocs.io/en/latest/concept_models/vivarium_iv_iron/child_model/concept_model.html)

### Verification and Testing
Ran a simulation for South Asia and checked that population and mortality looked reasonable.

### CODE REVIEW

**Review by albrja — APPROVED** [review]

File: src/vivarium_gates_child_iv_iron/model_specifications/south_asia.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/model_specifications/south_asia.yaml]: only one yaml file

File: src/vivarium_gates_child_iv_iron/model_specifications/south_asia.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+components:
+    vivarium_public_health:
+        population:
+            - BasePopulation()
+            - Mortality()
+
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/model_specifications/south_asia.yaml]: Need Fertility Component (see CIFF)

File: src/vivarium_gates_child_iv_iron/model_specifications/south_asia.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+configuration:
+    input_data:
+        input_draw_number: 0
+        location: 'South Asia'
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/model_specifications/south_asia.yaml]: test this with the location key removed.

**Review by rmudambi — CHANGES_REQUESTED** [review]

File: src/vivarium_gates_child_iv_iron/model_specifications/iv_iron_child.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    vivarium_public_health:
+        population:
+            - BasePopulation()
+            - Mortality()
+            - FertilityCrudeBirthRate()
+
+configuration:
+    input_data:
+        input_draw_number: 0
+        location: 'South Asia'
```
Comment (rmudambi) [inline src/vivarium_gates_child_iv_iron/model_specifications/iv_iron_child.yaml]: You can remove the location key here.

**Review by rmudambi — APPROVED** [review]

**Pull request merged**